src/libs/SuggestionUtils.ts

```diff
@@ -2,11 +2,10 @@ import CONST from '@src/CONST';
 
 /**
  * Return the max available index for arrow manager.
- * @param {Number} numRows
- * @param {Boolean} isAutoSuggestionPickerLarge
- * @returns {Number}
+ * @param numRows
+ * @param  isAutoSuggestionPickerLarge
```

Comment:
You can remove params without a description
```suggestion
```

Reply:
Done